console/web3ext/web3ext.go

```diff
@@ -477,6 +477,15 @@ web3._extend({
 			name: 'getSpamThrottlerCandidateList',
 			call: 'admin_getSpamThrottlerCandidateList',
 		}),
+		new web3._extend.Method({
+			name: 'syncStakingInfo',
+			call: 'admin_syncStakingInfo',
+			params: 3,
+		}),
+		new web3._extend.Method({
+			name: 'syncStakingInfoStatus',
+			call: 'admin_syncStakingInfoStatus',
+		}),
 	],
```

Comment:
In terms of the interface (rpc, caver), do we need to add the above 2 APIs?

Reply:
This change added 2 APIs to rpc, and this doesn't need to be added to caver at this moment.

Reply:
Yes, I understand
Thank you for your reply.